src/getHumanAge.test.js

```diff
@@ -1,10 +1,60 @@
 'use strict';
 
-describe('getHumanAge', () => {
+describe(`getHumanAge`, () => {
   const { getHumanAge } = require('./getHumanAge');
 
-  test('should be declared', () => {
+  test(`should be declared`, () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test(`should return 0 if both inputs <15`, () => {
+    expect(getHumanAge(14, 14))
+      .toEqual([0, 0]);
+  });
+
+  test(`should return 1 if both inputs are 15`, () => {
+    expect(getHumanAge(15, 15))
+      .toEqual([1, 1]);
+  });
+
+  test(`should return 1 if inputs are 15-23`, () => {
+    expect(getHumanAge(17, 19))
+      .toEqual([1, 1]);
+  });
+
+  test(`should return 1 if inputs are 23`, () => {
+    expect(getHumanAge(23, 23))
+      .toEqual([1, 1]);
+  });
+
+  test(`should return 2 if inputs are 24`, () => {
```

Comment:
The test description should say 'should return 2 if inputs are 24-28', as the cat's age in human years will change at 28. 🐱

Reply:
a separate check for 24 was requested by another mentor earlier, and the test for next 4 years is next